src/vivarium_nih_us_cvd/constants/metadata.py

```diff
@@ -11,14 +11,63 @@
 CLUSTER_PROJECT = "proj_simscience_prod"
 
 CLUSTER_QUEUE = "all.q"
-MAKE_ARTIFACT_MEM = "10G"
-MAKE_ARTIFACT_CPU = "1"
-MAKE_ARTIFACT_RUNTIME = "3:00:00"
+MAKE_ARTIFACT_MEM = 5  # GB
+MAKE_ARTIFACT_CPU = 1
+MAKE_ARTIFACT_RUNTIME = "4:00:00"
```

Comment:
Does it seriously take up to 4 hours to make an artifact?

Reply:
Yeah, it took 3.5 for Alabama

Reply:
Wow. All righty then.